src/rules/converters/eslint-plugin-react/jsx-ban-elements.ts

```diff
@@ -0,0 +1,38 @@
+import { RuleConverter } from "../../converter";
+
+export const convertJsxBanElements: RuleConverter = (tslintRule) => {
+    return {
+        rules: [
+            {
+                ...collectArguments(tslintRule.ruleArguments),
+                ruleName: "react/forbid-elements",
+            },
+        ],
+        plugins: ["eslint-plugin-react"],
+    };
+};
+
+const collectArguments = (ruleArguments: any[]) => {
+    if (ruleArguments.length === 0) {
+        return undefined;
+    }
+
+    const forbidArr: any[] = [];
```

Comment:
I think it should be a string type since you can have several items being the regex to match as the first item and the second one is the optional explanation as stated at https://github.com/palantir/tslint-react/blob/0ebaba30ac199643c43f5eb567ba65c038bfd8b4/src/rules/jsxBanElementsRule.ts#L26

Reply:
Thanks @KingDarBoja for reviewing. Yes, i agree 2nd item is optional here, thats why i have added the length check. 
I am making adding custom object based on the the input length in forbidArr, so i am not sure why it should be string type. Let me know if i am missing anything.

Reply:
It is explained on the above link:

```
options: {
            type: "list",
            listType: {
                type: "array",
                items: { type: "string" },
                minLength: 1,
                maxLength: 2,
            },
        },
```

Also there is an example:

`optionExamples: [[true, ["Object", "Use {} instead."], ["String"]]],`

Based on that, seems to be an array of array of strings :)

Reply:
Indeed, a `[string, string][]` would be a more accurate type definition.

Reply:
Here's an example config from the source project: https://github.com/palantir/tslint-react/blob/0ebaba30ac199643c43f5eb567ba65c038bfd8b4/src/rules/jsxBanElementsRule.ts